contracts/lib/IdentityLib.sol

```diff
@@ -400,6 +404,12 @@ library IdentityLib {
         self.rootsByState[state] = roots;
     }
 
+    function checkClaimExistence(Data storage self, uint256 hashIndex) internal view {
```

Comment:
Please use `_checkClaimExistence` and move the internal function after all public and external functions according to the conventions.